### DISCUSSION TITLE
Particularly interested in motoko internship

### DISCUSSION THREAD

**Discussion by panch8** [dfinity/motoko#4684] [General]
Hey everybody! I have just seen a post from @luc-blaeser in LinkedIn talking about a motoko internship which I am particularly interested in, because I could not find any contact channel I am creating this post here, hope do not disturb, or maybe help gather others in the same condition to interact in a single point of communication here. Would you share some more information? Thank you in anticipate.

**Comment by luc-blaeser** [comment]
Hi Francisco,

Thanks for reaching out! It would be about Motoko's upcoming enhanced
orthogonal persistence (https://github.com/dfinity/motoko/pull/4193) and
complex upgrade scenarios. The focus would be initially both on Motoko
programming and documentation, with potentially more advanced tasks later.

If you are interested you can send me your LinkedIn profile link, such that
I can forward it internally.

Best wishes
Luc

**Reply by panch8** [reply]
Thank you for the response. Sure this is my linkedin profile link [Me](https://www.linkedin.com/in/francisco-codino-43556621/). It is a great opportunity to Deep dive in this topics. 
Hope you have a good day

**Comment by luc-blaeser** [comment]
Thank you very much for the link. I forwarded your profile internally and
you'll hear back soon.
